samples/cpp/benchmark_app/main.cpp

```diff
@@ -792,6 +792,11 @@ int main(int argc, char* argv[]) {
                                               FLAGS_scale_values,
                                               FLAGS_mean_values,
                                               compiledModel.inputs());
+
+            batchSize = get_batch_size(app_inputs_info.at(0));
+            warn_if_no_batch(app_inputs_info.at(0));
+            slog::info << "Model batch size: " << batchSize << slog::endl;
```

Comment:
Should python be updated the same way?

Reply:
If it is possible, yeah, I think it is a good idea.